.github/workflows/reusable_pull_request.yml

```diff
@@ -83,4 +91,39 @@ jobs:
 
         for target in "${targets[@]}"; do
           swift package plugin generate-documentation --target "$target" --warnings-as-errors --analyze --level detailed
-        done
\ No newline at end of file
+        done
+
+  acceptable-language-check:
+    name: Acceptable language check
+    if: ${{ inputs.acceptable_language_check_enabled }}
+    runs-on: ubuntu-latest
+    timeout-minutes: 1
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+    - name: Run acceptable language check
+      shell: bash
+      env:
+        UNACCEPTABLE_LANGUAGE_PATTERNS: ${{ inputs.acceptable_language_check_word_list}}
+      run: |
+        set -euo pipefail
+
+        log() { printf -- "** %s\n" "$*" >&2; }
+        error() { printf -- "** ERROR: %s\n" "$*" >&2; }
+        fatal() { error "$@"; exit 1; }
+
+        test -n "${UNACCEPTABLE_LANGUAGE_PATTERNS:-}" || fatal "UNACCEPTABLE_LANGUAGE_PATTERNS unset"
+      
+        log "Checking for unacceptable language..."
+        UNACCEPTABLE_LANGUAGE_LINES=$(git -C . grep \
```

Comment:
variables in shell scrips ought to be lower case. ALL_CAPS variable names are
- for reserved shell variables (such as `PATH`, `EDITOR`, `SHELL`, `IFS`, ...)
- for adjusting _other_ programs' behaviour

> The name space of environment variable names containing lowercase letters is reserved for applications. Applications can define any environment variables with names from this name space without modifying the behavior of the standard utilities.

c.f. https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap08.html